recipes/nest/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+export MPI_FLAGS=--allow-run-as-root
+
+if [ $(uname) == Linux ]; then
+    export MPI_FLAGS="$MPI_FLAGS;-mca;plm;isolated"
+fi
+
+export CFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX \
+      -Dwith-mpi=OFF\
+      -Dwith-openmp=OFF \
+      -Dwith-python=3 \
```

Comment:
If you want to support Python 2 and 3:
```suggestion
      -Dwith-python=$(expr $PY3K + 2) \
```

Reply:
We do not support python2 any more